cypress/e2e/theme/theme.cy.js

```diff
@@ -40,4 +40,11 @@ describe('Check theme functionality', () => {
     cy.rendererApi(rendApi.id.setDarkTheme, 'false')
     cy.get('body').should('have.class', 'body--light')
   })
+  it('Zigbee mode', () => {
```

Comment:
From next time add a description on what the test is testing. Difficult to understand what this might be testing in the first go.